app/controllers/application_controller.rb

```diff
@@ -1,5 +1,19 @@
 class ApplicationController < ActionController::Base
-  def current_user
-    User.first
+  helper_method :current_user
```

Comment:
- [ ] The `current_user` method in `ApplicationController` should be  removed. Devise provides a method to do this.